### PR TITLE
#1 Role update and support for Debian 11

### PR DESCRIPTION
**Pull request by ablancolopez**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;
* Add "idealista/benders" as reviewer

### Description of the Change

Update the role to the latest versions of its dependencies and provide support for Debian 11 and dotnet 6.0.x.

### Benefits

Support for Debian 11 and newer ansible stack versions.

### Possible Drawbacks

Drop support for Debian 9

### Applicable Issues

#1

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ galaxy_info:
   author: frantsao
```
Comment (frantsao) [inline meta/main.yml]: Please change author to "idealista"

File: test-requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
-ansible==2.9.19
-molecule==3.0.8
-docker==4.4.4
-ansible-lint==5.0.6
\ No newline at end of file
+ansible==5.2.0
+molecule==3.4.0
```
Comment (frantsao) [inline test-requirements.txt]: Some of these packages can be upgraded, at least molecule, docker. With that upgrades, you will also need molecule-docker package, and maybe some changes at molecule.yml file.

File: test-requirements.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
+docker==5.0.0
+ansible-lint==5.3.2
+rich==*
```
Comment (frantsao) [inline test-requirements.txt]: Please fix rich to a concrete version.

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,14 +5,13 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased](https://github.com/idealista/dotnetcore_role/tree/develop)
 ### Changed
+- *[#1](https://github.com/idealista/dotnetcore_role/issues/1) General role update* @blalop
+
 ### Added
+- *[#1](https://github.com/idealista/dotnetcore_role/issues/1) Debian 11 support* @blalop
 ### Removed
+- *[#1](https://github.com/idealista/dotnetcore_role/issues/1) Debian 9 support* @blalop
```
Comment (frantsao) [inline CHANGELOG.md:13]: I would add here "BREAKING" :-)

**Review by frantsao — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (ablancolopez) [reply CHANGELOG.md:13]: Will do when updating the changelog to reflect the new major release :)

**Review by jperera — APPROVED** [review]

**Review by santi-eidu — APPROVED** [review]

**Pull request merged**